scipy/optimize/tests/test_linprog.py

```diff
@@ -1367,51 +1359,82 @@ def test_network_flow(self):
 
 
 class TestLinprogSimplexDefault(LinprogSimplexTests):
-    options = {}
 
-    def test_bug_7237(self):
-        # we want to check for the error
+    def setup_method(self):
+        self.options = {}
+
+    def test_bug_5400(self):
+        with pytest.raises(ValueError):
+            super(TestLinprogSimplexDefault, self).test_bug_5400()
+
+    def test_bug_7237_low_tol(self):
+        # Fails if the tolerance is too strict. Here we test that
+        # even if the solutuion is wrong, the appropriate error is raised.
+        self.options.update({'tol': 1e-12})
         with pytest.raises(ValueError):
             super(TestLinprogSimplexDefault, self).test_bug_7237()
 
-    def test_bug_8174(self):
-        # we want to check for the warning about the pivot
+    def test_bug_8174_low_tol(self):
```

Comment:
I see that `test_bug_8174` and `test_bug_8174_2` now pass, addressing that comment in #9268. So I think this PR closes #9268.

Reply:
I agree #9268 can be closed.